.storybook/sourcecode-addon/storybookPanel.jsx

```diff
@@ -35,6 +35,13 @@ const SourceCodePanel = props => {
 
   const handleToggleCompiled = () => setShowCompiled(!showCompiled);
 
+// Update page title on load or tab change
+  useEffect(() => {
+    const title = api.getCurrentStoryData()?.kind + ' ' + api.getUrlState()?.viewMode
```

Comment:
```suggestion
    const title = api.getCurrentStoryData()?.kind + ' ' + api.getUrlState()?.viewMode;
```
A simpler fix for the missing semicolon in the case you don't like the other suggestion.